.github/workflows/stale-bot.yml

```diff
@@ -0,0 +1,39 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+name: 'Close stale issues and PR '
+on:
+  schedule:
+    - cron: '0 20 * * SUN' # every Sunday at 20 am UTC: PST 0:00 AM "
+    #- cron: '* * * * *' # For every minute and only for testing purpose"
```

Comment:
We can delete this. No point in having it here for testing purposes once it gets merged.

Reply:
Yes. I intend to delete it after discussing in the standup to know more about our standard of marking staling and closing.